inst/.gitignore

```
@@ -0,0 +1,3 @@
+.env
+*.log
+*.exe
```

My inner perfectionist says that it is better to make all the lines in the same format

eg

```suggestion
.env
.log
.exe
```

or 

```suggestion
*.env
*.log
*.exe
```